api/config/AWS-IOT/README.md

```diff
@@ -0,0 +1,6 @@
+PLEASE ADD THE FOLLOWING FILES TO THIS FOLDER TO CONNECT TO AWS IOT AND TO SUCCESSFULLY RUN THE RFID SERVER: 
+
+- AmazonRootCA1.pem
+- cert.pem.crt
+- private.pem.key
+- public.pem.key
```

Comment:
lets add this to the [Getting Started](https://github.com/SCE-Development/Core-v4/wiki/Getting-Started) section on the Core-v4 wiki instead